pom.xml

```diff
@@ -88,6 +88,7 @@
                     <plugin id="org.eclipse.equinox.launcher.gtk.linux.ppc64le" />
                     <plugin id="org.eclipse.equinox.launcher.gtk.linux.aarch64" />
                     <plugin id="org.eclipse.equinox.launcher.gtk.linux.x86_64" />
+                    <plugin id="org.eclipse.equinox.launcher.win32.win32.aarch64" />
```

Comment:
I'm not 99% sure this one is fine.

Reply:
I think Tycho would break if it cannot find a plug-in as part of a feature cc @laeubi

Reply:
This is exclusion so most probbaly should work.